service-front/module/Application/src/Services/SiriusApiService.php

```diff
@@ -160,15 +160,20 @@ public function abandonCase(array $data, Request $request): array
         ];
     }
 
-    /**
-     * @param string $base64suffix
+     /**
      * @param array $caseDetails
+     * @param string $systemType
      * @param Request $request
+     * @param string $base64suffix
      * @return array
      * @throws GuzzleException
      */
-    public function sendPostOfficePDf(string $base64suffix, array $caseDetails, Request $request): array
-    {
+    public function sendPdf(
```

Comment:
i've changed this to be a more generic function that can be used to send both the post-office and vouching letters since theres a significant overlap.
Having said that i'm not 100% sure what the significance of the `$base64suffix` parameter is - does its absence from the vouching letter mean its not a PDF? Does that mean my function name doesn't make sense? Should I actually keep them as 2 functions?

Reply:
You've handled that spot on: it's an optional parameter of an existing PDF file in base-64 format. If you provide it, Sirius will generate the letter then stitch the `base64suffix` afterwards. In Yoti context, this is us sending a PDF that they've generated with a custom cover letter.

Might be worth renaming that variable to something like `pdfSuffixBase64` to be a bit clearer, and maybe updating the `@param` to give quick explanation of what it's for.